achievements/api-platform.md

```diff
@@ -1 +1,49 @@
+# API Platform Project achievements
 
+This weekend we were multiple contributors to API Platform, some of the core team were here too to help in the hackathon.
+When it comes to **API Platform** we just merge 20 PR, 10 new opened PR, 87 closed issues and 4 news. Lots of word being done from the old and the new contributors, big kudos !!
+
+Since 2 years from now, API Platform supports subresources, this features does give access to include relation in the relation, this feature is used a lot and is very interesting, but since it's has alot of complexity and limited.
+
+With the help of @vincentchalamon, @soyuka & @torreytsui, we made some research on how to improve the subresources. How to reduce the complexity of the implementation, the management of collection is very important for us.
+
+Thanks to @jewome62, it's now possible to populate an object recursively with the serializer, to normalize a DateTimeZone object, and to get an instanciated object from an associative array of values.
+
+@soyuka did a lot to close stale issues, along with @vincentchalamon.
+
+@gregoirehebert fixed the fact the cache was still purged even if there was no get operation on an ApiResource. and bring you a way to switch version for the documentation.
```

Comment:
```suggestion
@gregoirehebert fixed the fact the cache was still purged even if there was no get operation on an ApiResource, and brings you a way to switch version for the documentation.
```

Reply:
`get` maybe ?
"brings you a way to switch version for the documentation" needs some explanation I think.